app/jobs/dttp/retrieve_award_job.rb

```diff
@@ -19,9 +19,8 @@ def perform(trainee, timeout_after = nil)
         return
       end
 
-      awarded = RetrieveAward.call(trainee: trainee)
-
-      if awarded
+      if awarded?
+        trainee.awarded_at = award_status["dfe_qtseytsawarddate"]
         trainee.award!
```

Comment:
Could we make this into a method that takes the date as an argument like we've done with `Trainee#trn_received!`

Reply:
Ah, great, thanks, I was looking for similar calls in the other events, but I handn't looked around enough :).